docs/website/docs/v1.x/dsl/server.md

```diff
@@ -98,15 +98,17 @@ object HelloWorldAdvanced extends App {
+| `Server.maxRequestSize(8 * 1024)`                           | handle max request size of 8 KB (default 4 KB)                                                                                                                                                            |
+| `Server.port(portNum)` or `Server.bind(portNum)`            | Bind server to the port, refer to examples above                                                                                                                                                          |
+| `Server.ssl(sslOptions)`                                    | Creates a new server with ssl options. [HttpsHelloWorld](https://github.com/dream11/zio-http/blob/main/example/src/main/scala/example/HttpsHelloWorld.scala)                                              |
+| `Server.acceptContinue`                                     | Sends a [100 CONTINUE](https://www.w3.org/Protocols/rfc2616/rfc2616-sec8.html#sec8.2.3)                                                                                                                   |
+| `Server.disableFlowControl`                                 | Refer [Netty FlowControlHandler](https://netty.io/4.1/api/io/netty/handler/flow/FlowControlHandler.html)                                                                                                  |
+| `Server.disableLeakDetection`                               | Disable any leak detection Refer netty's [ResourceLeakDetector](https://netty.io/4.0/api/io/netty/util/ResourceLeakDetector.Level.html)                                                                   |
+| `Server.simpleLeakDetection`                                | Simplistic leak detection comes with small over head. Refer netty's [ResourceLeakDetector](https://netty.io/4.0/api/io/netty/util/ResourceLeakDetector.Level.html)                                        |
+| `Server.paranoidLeakDetection`                              | Comes with highest possible overhead (for testing purposes only). Refer netty's [ResourceLeakDetector](https://netty.io/4.0/api/io/netty/util/ResourceLeakDetector.Level.html)                            |
+| `Server.consolidateFlush`                                   | Flushing content is done in batches. Can potentially improve performance.                                                                                                                                 |
+| `Server.requestDecompression(strict)`                       | Creates a Server with support for Request Decompression. If `strict` is set to `true` it uses strict handling of deflate(if used), otherwise handles it in a more lenient fashion.                        |
+| `Server.responseCompression(contentSizeThreshold, options)` | Creates a Server with support for Response Compression. You can enable compression for all responses by setting `contentSizeThreshold` to 0. `options` should be an `IndexedSeq` of `CompressionOptions`. |
```

Comment:
add separator

Reply:
It's already there. Doesn't look like it because of the addition.

Reply:
<img width="1360" alt="image" src="https://user-images.githubusercontent.com/23661702/156696698-b1c0b5ee-dd74-4c52-be4e-b9422e66960a.png">
No separator

Reply:
<img width="1074" alt="Screenshot 2022-03-15 at 1 00 57 PM" src="https://user-images.githubusercontent.com/21025968/158328022-c6ad4508-5354-4bad-8ca9-87817d013a59.png">

It's because of the diff green colour. I checked the file, it's there.